app/helpers/api/ordapi/postOrderItemBulk.js

```diff
@@ -27,7 +27,7 @@ const formatPostData = ({
   price: parseFloat(formData.price),
 }));
 
-const getPostOrderItemEndpoint = (orderId) => `${orderApiUrl}/api/v1/orders/${orderId}/order-items/batch`;
+const getOrderItemEndpoint = (orderId, itemId) => `${orderApiUrl}/api/v1/orders/${orderId}/order-items/${itemId}`;
```

Comment:
`itemId` is actually the catalogue item ID now (e.g. 10001-001), so it might be better to rename this to `catalogueItemId` just to avoid any confusion around people thinking it is an arbitrary order item ID (e.g. 1, 2, 3 etc.).